output/pursuits.json

```diff
@@ -0,0 +1,3692 @@
+{
+  "1134691": {
+    "Place": [975684424]
+  },
+  "5697712": {
+    "Place": [975684424]
+  },
+  "13409814": {
+    "ItemCategory": [5]
+  },
+  "14014810": {
+    "ActivityMode": [2319502047]
+  },
+  "17342954": {
+    "Place": [3526908984]
+  },
+  "19656477": {
+    "ActivityMode": [1826469369],
+    "ItemCategory": [8]
+  },
+  "22608947": {
+    "ActivityMode": [1848252830]
+  },
+  "30972315": {
+    "ActivityMode": [2394616003],
+    "DamageType": [1847026933]
+  },
+  "31218063": {
+    "ItemCategory": [6]
+  },
+  "36120323": {
+    "ActivityMode": [2319502047]
+  },
+  "39386534": {
+    "ActivityMode": [1848252830]
+  },
+  "42076730": {
+    "Place": [3526908984]
+  },
+  "46102161": {
+    "ActivityMode": [1848252830, 1164760504]
+  },
+  "46155327": {
+    "ActivityMode": [1418469392]
+  },
+  "47910874": {
+    "ActivityMode": [1264443021]
+  },
```

Comment:
The word Mayhem needs to be added to the `crucible` detector

Reply:
The `ActivityMode` definition is really cool - it defines a hierarchy of activity types. My plan is to leverage this in the Progress page. So if you choose Mayhem as your activity, all general Crucible bounties will light up. But if you choose Crucible as your activity, Mayhem bounties won't necessarily light up, because you have to be in Mayhem to progress those.